cmd/finch/support_bundle.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/lima"
+	"github.com/runfinch/finch/pkg/support"
+)
+
+func newSupportBundleCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleCommand := &cobra.Command{
+		Use:   "support-bundle",
+		Short: "Gathers logs and info into a bundle to help debug issues",
+	}
+	supportBundleCommand.AddCommand(
+		newSupportBundleGenerateCommand(logger, builder, lcc),
+	)
+	return supportBundleCommand
+}
+
+func newSupportBundleGenerateCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleGenerateCommand := &cobra.Command{
+		Use:   "generate",
+		Args:  cobra.NoArgs,
+		Short: "Generate support bundle",
+		RunE:  newGenerateSupportBundleAction(logger, builder, lcc).runAdapter,
+	}
+
+	supportBundleGenerateCommand.Flags().StringArray("include", []string{}, "additional files to include in the support bundle")
+	supportBundleGenerateCommand.Flags().StringArray("exclude", []string{}, "files to exclude from the support bundle")
```

Comment:
I guess the files to be excluded are deterministric? Should we list all the possible strings or give some examples here? I don't know what format of string to put here only from this description.

Reply:
Will you add examples?

Reply:
On second thought, I don't think we should. I think it may clutter up the description too much, and I think it's better to encourage trial and error.